### ISSUE TITLE
Design: Accesability issue with inputs without labels

### ISSUE THREAD

**Issue by KubaMikolajczyk** [Joystream/atlas#4178]
@bedeho @dmtrjsg
I'm raising the issue Bartosz brought me about the lack of accessibility when providing inputs without labels:
🎥 Loom explainer: https://www.loom.com/share/741b3f5a66834de8b8c67e60fdd609f1
📄 Figma exploration file: https://www.figma.com/file/C3hvYPVkZ7TcjAsXfqizSD/User-accounts?type=design&node-id=2242%3A56646&t=iRl0ti8IjCbeYnDu-1

**Comment by dmtrjsg** [comment]
Accessibility at our stage is not a strict requirement, so I think we shouldn't bother at this point.. I like the Twitter approach aesthetically but let's not go there at this point of time. 

cc @bedeho @drillprop

**Comment by drillprop** [comment]
I think that's also a UX issue, for example practically no one expects two email inputs in the first sign-up step. So a user could just start writing the password here because he didn't pay attention to what's in the placeholder. A label is more prominent, and there is a smaller chance for mistakes.  That's why I requested the change. 

I would just add a label above as we have in other places. I agree it looks slightly worse, but I think we can live with that.
And for the placeholder, I would use example input text like in the screen below: 

![image](https://github.com/Joystream/atlas/assets/51168865/40e4b357-b320-406a-92b1-9e9ea29dc5cd)

**Comment by dmtrjsg** [comment]
If you guys @drillprop @KubaMikolajczyk both are committed to this approach, let's go for it. This would not impact any onboarding metrics imho, so if rework is not required I don't mind. 

My point is that I don't think this is a huge issue and both ways are okay.

**Comment by KubaMikolajczyk** [comment]
No rework required it will look slightly worse but will be more usable if we at least add labels. So I agree with Bartosz here
